cdap-app-templates/cdap-etl/cdap-etl-core/src/main/java/co/cask/cdap/etl/batch/config/ETLBatchConfig.java

```diff
@@ -140,6 +140,15 @@ public int hashCode() {
     return Objects.hash(super.hashCode(), schedule, actions);
   }
 
+  @Override
+  public String toString() {
+    return "ETLBatchConfig{" +
+      "engine=" + engine +
+      ", schedule='" + schedule + '\'' +
+      ", actions=" + actions +
+      ", driverResources=" + driverResources +
+      "} " + super.toString();
```

Comment:
how does this print? two different things? ``ETLBatchConfig{.....}  ETLConfig{...}``?